### PR TITLE
fix broken link for ignored paths

### PR DESCRIPTION
**Pull request by antleblanc**
Correct a broken link on the readme file.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -154,7 +154,7 @@ Additional global variables your code accesses during execution.
 
 Type: `array`
 
-Some [paths](https://github.com/sindresorhus/xo/blob/4a0db396766118d7918577d759cacb05cd99a354/index.js#L14-L20) are ignored by default. Additional ignores can be added here.
+Some [paths](https://github.com/sindresorhus/xo/blob/master/index.js#L10-L19) are ignored by default. Additional ignores can be added here.
```
Comment (arthurvr) [inline readme.md]: > master

Better to use an actual commit hash as this way it's going to break when we add more code above that array definition.

**Comment by arthurvr** [comment]
Thanks for the PR!

**Comment by kevva** [comment]
Can you squash your commits :)?

**Comment by antleblanc** [comment]
Done!
Thanks for guidance

**Comment by kevva** [comment]
Thanks man!

**Pull request merged**